wtclient_client.go

```diff
@@ -0,0 +1,244 @@
+package lndclient
+
+import (
+	"context"
+	"time"
+
+	"github.com/lightningnetwork/lnd/lnrpc/wtclientrpc"
+	"google.golang.org/grpc"
+)
+
+// Implementation of the watchtower client interface
+// https://lightning.engineering/api-docs/category/watchtowerclient-service/
+type WatchtowerClientClient interface {
+	ServiceClient[wtclientrpc.WatchtowerClientClient]
+
+	AddTower(ctx context.Context, pubkey []byte, address string) error
```

Comment:
These methods are exported and we therefore require documentation here, especially in an interface. You could copy over the ones from `wtclient.proto`. It would also be good to make the implementation's docstrings consistent with the interface's, have a look into the other files how it was done.

Reply:
I've documented all the methods now, take a look!